package.json

```diff
@@ -737,45 +759,50 @@
           "when": "view == openshiftProjectExplorer && viewItem == component",
           "group": "c3@2"
         },
+        {
+          "command": "openshift.component.unlink",
+          "when": "view == openshiftProjectExplorer && viewItem == component",
+          "group": "c4"
```

Comment:
Keep Unlink command in c3 group as c3@3. This way it will appear in the same group with Link commands.